charts/trento-server/Chart.yaml

```diff
@@ -28,3 +28,7 @@ dependencies:
     version: ~6.36.1
     repository: https://grafana.github.io/helm-charts/
     condition: grafana.enabled
+  - name: rabbitmq
+    version: ~10.3.9
+    repository: https://charts.bitnami.com/bitnami/
+    condition: rabbitmq.enabled
```

Comment:
Thinking a bit more about this: could it make more sense to change the condition to `wanda.enabled` ? Then we wouldn't need to enable them separately (I don't see the advantage of being able to enable rabbitmq w/o wanda)

Reply:
I don't think it is a good idea. In the future, the dashboard will use rabbitmq as well, and we might want a more tailored deployments for testing.
Besides, you would like to use an externally deployed rabbitmq for some reason, and don't need the kubernetes one.
And finally, all the other components (like postgresql) follow this approach, so I like to have them working in the same way